src/app/states/azure-cognitive.service.ts

```diff
@@ -23,6 +28,7 @@ export interface AzureCognitiveSettings {
 export interface AzureCognitiveState {
   speechConfig: SpeechConfig | null;
   settings: AzureCognitiveSettings | null;
+  managed: boolean;
```

Comment:
somehow this does not really work on my machine. i see at the start both, invalid login and successfull. and after entering the email/password i can not press login.

<img width="619" alt="Screenshot 2023-05-08 at 18 58 55" src="https://user-images.githubusercontent.com/14818450/236884975-ea71f679-8cc9-46cd-9519-8d17184b734c.png">
<img width="515" alt="Screenshot 2023-05-08 at 18 59 13" src="https://user-images.githubusercontent.com/14818450/236884991-5dcf508f-8538-46ac-8717-5166b1af298c.png">